docs/build/compute/quickstart.md

```diff
@@ -0,0 +1,142 @@
+:::
+
+## Deploy Your First Workload
+
+Let's deploy a simple "Hello World" API service that runs in a secure Confidential VM.
+
+### Step 1: Create Your Docker Compose File
+
+Create a simple API service:
+
+```yaml
+services:
+  web:
+    image: caddy:2
+    command: |
+      caddy respond --listen :8080 --body '{"hello":"world"}' --header "Content-Type: application/json"
+```
+
+### Step 2: Create Workload
+
+Choose your preferred workload creation method. Both options create the same secure workload
+
+<Tabs>
+<TabItem value="ui" label="Create Workload with UI" default>
+
+Create a new workload with the [nilCC Workload Manager](https://nilcc.nillion.com) UI. This workload creation method is recommended for first-time nilCC users, visual workflow management, and ongoing monitoring.
+
+[![nilCC Create Workload](/img/nilcc-create-workload.png)](https://nilcc.nillion.com/workloads/create)
+
+1. **Visit**: [nilCC Workload Manager](https://nilcc.nillion.com)
+2. **Authenticate**: Enter your nilCC API Key to log in
+3. **Create New Workload**:
+   - **Name**: `hello-world-api`
+   - **Docker Compose**: Paste the YAML content from Step 1
+   - **Service to Expose**: `web` (automatically detected)
+   - **Port**: `8080` (automatically detected)
+   - **Resources**: Optionally adjust the resource tier as needed
+4. **Deploy**: Click "Create Workload" and monitor the deployment status in real-time
+
+</TabItem>
+<TabItem value="api" label="Create Workload with API">
+
+Create a new workload programatically with the nilCC API. This workload creation method is recommended for automation, CI/CD pipelines, and programmatic deployments.
+
+Update the command below with your nilCC API key, chosen resource tier (cpus, gpus, memory, and disk size), and the latest artifacts version:
+
+```bash
+curl -X POST https://api.nilcc.nillion.network/api/v1/workloads/create \
+  -H "x-api-key: YOUR_API_KEY" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "hello-world-api",
+    "dockerCompose": "services:\n  web:\n    image: caddy:2\n    command: |\n      caddy respond --listen :8080 --body '\''{\"hello\":\"world\"}'\'' --header \"Content-Type: application/json\"",
+    "serviceToExpose": "web",
+    "servicePortToExpose": 8080,
+    "cpus": YOUR_CPUS,
+    "memory": YOUR_MEMORY,
+    "disk": YOUR_DISK,
+    "gpus": YOUR_GPUS,
+    "artifactsVersion": LATEST_ARTIFACTS_VERSION
+  }'
+```
+
+**Key API Parameters:**
+
+- `dockerCompose`: Your Docker Compose file as an escaped string
+- `serviceToExpose`: Which service gets the public domain (must match a service name)
+- `servicePortToExpose`: Which port on that service to expose publicly
+- `artifactsVersion`: nilCC VM image version
+
+</TabItem>
+
+</Tabs>
+
+## Test Your Workload
+
+### Access Your Secure API within nilCC
+
+Once deployed, your workload gets a unique domain. Access it at:
+
+```bash
+# Your workload will be available at a domain like:
+curl https://[your-secure-workload].nillion.network
```

Comment:
For production all user workloads return a URL on the nillionusercontent.com domain.